eng/templates/steps/build.yml

```diff
@@ -35,10 +35,21 @@ steps:
     git diff
   displayName: Remove RunAsPolicy From MaestroApplication
 
+- powershell: |
+    [xml]$packagesProps = Get-Content -Path .\Directory.Packages.props
+    $aspireNode = $packagesProps.Project.ItemGroup.PackageVersion | Where-Object { $_.Include -eq 'Aspire.Hosting' }
+    $fileContent = @"
+    {
+      `"microsoft.net.sdk.aspire`": `"$($aspireNode.Version)/8.0.100`"
+    }
+    "@
+    $fileContent | Out-File -FilePath $(Agent.TempDirectory)\aspire.json
+  displayName: Create Aspire Rollback File
+
 - powershell: |
     . .\eng\common\tools.ps1
     InitializeDotNetCli -install:$true
-    .\.dotnet\dotnet workload install aspire
+    .\.dotnet\dotnet workload install aspire --from-rollback-file $(Agent.TempDirectory)\aspire.json
```

Comment:
Is this the only way to pin a workload version? Seems overly complicated

Reply:
yup, `dotnet workload install` doesn't support anything else

Reply:
It is what it is then.

Could you please include a small comment explaining why this is done? Thanks!